### PR TITLE
updated screen shots to match the new ui

### LINKED ISSUE #1490: Controlling access to the Admin Console chapter
### Description

Update this Server Administration Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1490

### CODE REVIEW

File: server_admin/topics/admin-console-permissions/fine-grain.adoc [hunk 1]
```diff
@@ -43,20 +43,20 @@ The first thing we must do is login to the Admin Console so we can set up permis
 of the client, we want to define fine-grain permissions for.
 
 .Client management
-image:{project_images}/fine-grain-client.png[]
+image:{project_images}/fine-grain-client.png[Fine grain client]
```
Comment (andymunro) [inline server_admin/topics/admin-console-permissions/fine-grain.adoc:46]: Text on this screen is hard to read.  Can you shrink this screen by a quarter and recapture it?

File: server_admin/topics/admin-console-permissions/fine-grain.adoc [hunk 2]
```diff
@@ -43,20 +43,20 @@ The first thing we must do is login to the Admin Console so we can set up permis
...
+image:{project_images}/fine-grain-client-permissions-tab-off.png[Fine grain client permissions tab]
 
 By default, each client is not enabled to do fine grain permissions.  So turn the `Permissions Enabled` switch to on
 to initialize permissions.
 
 IMPORTANT: If you turn the `Permissions Enabled` switch to off, it will delete any and all permissions you have defined for this client.
 
 .Client permissions tab
-image:{project_images}/fine-grain-client-permissions-tab-on.png[]
+image:{project_images}/fine-grain-client-permissions-tab-on.png[Fine grain permission tab]
```
Comment (andymunro) [inline server_admin/topics/admin-console-permissions/fine-grain.adoc:59]: Text on this screen is hard to read.  Can you shrink this screen by a quarter and recapture it?

**Review by andymunro — CHANGES_REQUESTED** [review]
Only two screen shot changes to suggest.

File: server_admin/topics/admin-console-permissions/fine-grain.adoc (hunk 2 above)
Comment (andymunro) [reply server_admin/topics/admin-console-permissions/fine-grain.adoc:59]: The others look good, but this one could still be resized.

File: server_admin/topics/admin-console-permissions/fine-grain.adoc [hunk 3]
```diff
@@ -121,26 +121,26 @@ and not perform any other types of user administration.
...
 .View leads role permission tab
-image:{project_images}/fine-grain-view-leads-role-tab.png[]
+image:{project_images}/fine-grain-view-leads-role-tab.png[Fine grain view leads role]
 
 If we click on that tab and turn the `Permissions Enabled` on, you'll see that there
 are a number of actions we can apply policies to.
 
 .View leads permissions
-image:{project_images}/fine-grain-view-leads-permissions.png[]
+image:{project_images}/fine-grain-view-leads-permissions.png[Fine grain view leads permissions]
```
Comment (andymunro) [inline server_admin/topics/admin-console-permissions/fine-grain.adoc:137]: This one could also benefit from being resized.

File: server_admin/topics/admin-console-permissions/fine-grain.adoc [hunk 4]
```diff
@@ -186,7 +186,7 @@ by a client.  If we log back into the admin console to our master realm admin an
   to the `sales-application` permissions page, you'll see the `map-roles` permission.
 
 .Client map-roles permission
-image:{project_images}/fine-grain-client-permissions-tab-on.png[]
+image:{project_images}/fine-grain-client-permissions-tab-on.png[Fine grain client permissions]
```
Comment (andymunro) [inline server_admin/topics/admin-console-permissions/fine-grain.adoc:189]: This one could also be resized to be more legible.

**Review by andymunro — CHANGES_REQUESTED** [review]
@edewit A few more screens to resize to make the text legible. Otherwise, looks good.

**Comment by andymunro** [comment]
Thanks, @edewit.  Looks good, but I still see two screens not showing up as being resized:

- fine-grain-client-permissions-tab-on.png
- fine-grain-users-permissions.png

**Comment by edewit** [comment]
sorry, updated those now... @andymunro

**Comment by andymunro** [comment]
Thanks for the quick update, @edewit.  Looks perfect.
Another PR ready to merge, @ssilvert

**Review by andymunro — APPROVED** [review]

**Review by andymunro — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**